clean-up-migrations.md

```diff
@@ -0,0 +1,126 @@
+# Clean up your migrations!
+
+Database migrations are a great way to safely update your database schema,
+and so it's very useful for production where you do not want to lose the data
+you have. If. you want to know more about migrations - SymfonyCasts has a
+[screencasts](https://symfonycasts.com/screencast/symfony-doctrine/migrations)
+about it, check it out!
+
+But here, I will give you a tip on how to clean up your migrations correctly.
+First of all, this is mostly for local development, because usually it's not
+important how many migrations you have on production - all those migrations
+were already executed.
+
+But in local development it's also a good idea to create the database and run
+the migrations so that you have exactly what's on production and a `migration_versions`
+table that matches production instead of simply running `symfony console doctrine:schema:update`
+command.
+
+And that's when many migration files may cause some performance issues locally.
+In this case, you can drop all the migrations you have so far and create a single
+migration that will recreate the whole database schema from scratch.
```

Comment:
I still don't think we've explained the problem clearly enough. I think we're closer. It should be something like this:

A) Locally, we recommend setting up your local system with `doctrine:migrations:migrate` instead of `doctrine:schema:update` so that you can easily test any new migrations you add.

B) But, if you have a lot of old migration files, this can start to get kinda slow! On Symfonycasts, we had 50 (fill in real number).

C) And so, while it's TOTALLY optional, if a giant `migrations/` folder bothers you, you CAN clean it up.

So, I think just some light rephrasing of these few paragraphs :)